allure-generator/src/test/javascript/components/tree.spec.js

```diff
@@ -116,12 +116,12 @@ describe('Tree', function () {
         ]
     });
 
-    describe('empty data', function () {
-        beforeEach(function () {
+    describe('empty data', function() {
```

Comment:
.... and make all nested as arrow.

So we will get one `describe('...', function() {})` per file.